frame/balances/src/tests.rs

```diff
@@ -342,7 +342,7 @@ macro_rules! decl_tests {
 		fn balance_works() {
 			<$ext_builder>::default().build().execute_with(|| {
 				let _ = Balances::deposit_creating(&1, 42);
-				System::assert_has_event(Event::Balances(crate::Event::Deposit(1, 42)));
+				System::assert_has_event(Event::Balances(crate::Event::Deposit{who: 1, amount: 42}));
```

Comment:
```suggestion
				System::assert_has_event(Event::Balances(crate::Event::Deposit{ who: 1, amount: 42 }));
```
Same change as requested in https://github.com/paritytech/substrate/pull/9993/files#r739342924 . Goes for everything in this macro